_includes/master/manifests/rbac.yaml

```diff
@@ -73,8 +73,11 @@ rules:
     resources:
       - nodes/status
     verbs:
+      # Needed for clearing NodeNetworkUnavailable flag (/node PR#2269)
       - patch
 {%- if include.datastore == "kdd" %}
+      # Calico uses UpdateStatus instead of Update on nodes (/libcalico-go PR#971)
```

Comment:
I'd focus on what this permissions is used for, not status vs not.

e.g. "   # Calico stores some configuration information in node annotations."

Reply:
And I'd drop the PR numbers - this is nominally user-facing, so they probably care less about PRs and more about function.

Reply:
Yeah, I was trying to find something to point to for context. Maybe when we get some more docs on the subject, we can point to that later on.